src/java.base/share/native/libjava/ClassLoader.c

```diff
@@ -23,16 +23,17 @@
  * questions.
  */
 
-#include <stdlib.h>
 #include <assert.h>
+#include <errno.h>
```

Comment:
Whats errno.h for?

Reply:
Thanks for catching that. It was for [the other PR](https://github.com/openjdk/jdk/pull/7829).